webapp/package.json

```diff
@@ -5,6 +5,7 @@
     "main": "src/index.js",
     "scripts": {
         "build": "webpack --mode=production --display-error-details --progress",
+        "debug": "webpack --mode=development --display-error-details --progress",
```

Comment:
The starter template uses a different command. Should we adopt what the starter template does here?